include/zephyr/arch/arc/v2/arcv2_irq_unit.h

```diff
@@ -199,9 +199,11 @@ bool z_arc_v2_irq_unit_is_in_isr(void)
 /**
  * @brief Sets an IRQ line to level/pulse trigger
  *
- * Sets the IRQ line <irq> to trigger an interrupt based on the level or the
- * edge of the signal. Valid values for <trigger> are _ARC_V2_INT_LEVEL and
+ * Sets the IRQ line @p irq to trigger an interrupt based on the level or the
+ * edge of the signal. Valid values for @p trigger are _ARC_V2_INT_LEVEL and
```

Comment:
Shouldn't we bring back these directly to the main branch instead of the collab branch if these tags are wrong?

Reply:
they are already, there is a PR https://github.com/zephyrproject-rtos/zephyr/pull/78387